webpack/assets/javascripts/services/ChartService.js

```diff
@@ -1,76 +1,57 @@
 import uuidV1 from 'uuid/v1';
+import Immutable from 'seamless-immutable';
 import { donutChartConfig, donutLargeChartConfig } from './ChartService.consts';
 
 const sizeConfig = {
   regular: donutChartConfig,
   large: donutLargeChartConfig,
 };
 
-function getChartConfig({
-  data, config, onclick, id = uuidV1(),
-}) {
-  if (!data) {
-    return {};
+const isDataExist = (data) => {
+  if (!data || data.length === 0) {
+    return false;
```

Comment:
This method is the whole reason I don't need unloadData in https://github.com/theforeman/foreman_remote_execution/pull/130 - `nonEmptyData` used to remove data, so that

`[['sucess', 3], ['failed', 2],['pending', 0], ['canceled', 0]]` turned into `[['sucess', 3], ['failed', 2]]`. 
This is what caused the problem when rendering and `unloadData=false`. However, this function now does not remove anything. So `[['sucess', 3], ['failed', 2],['pending', 0], ['canceled', 0]]` is sent to the DonutChart like that, and it renders fine. I'm not even sure what was the purpose of `nonEmptyData`, so I think it's probably ok to keep it.